com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/internal/spi/SpiFrameHandler.java

```diff
@@ -522,7 +524,8 @@ private synchronized boolean sendNextFrame() {
                 return false;
             }
         }
-
+        int sequenceNumber = nextEzspSequenceNumber.getAndIncrement() & 0xff;
```

Comment:
Since ```sequenceNumber``` doesn't appear to be used at all (other than in the next line) it might be better not to create this variable and directly set the sequence number?

```
nextFrame.setSequenceNumber(nextEzspSequenceNumber.getAndIncrement() & 0xff);
```